### PR TITLE
move community code to status-im2

### PR DESCRIPTION
**Pull request by flexsurfer**
move community code to status-im2
remove outdated code (import and create community)

### CODE REVIEW

**Review by Parveshdhull — APPROVED** [review]

**Review by J-Son89 — APPROVED** [review]

**Comment by flexsurfer** [comment]
image

**Comment by flexsurfer** [comment]
3 community tests definitely related :(

**Comment by flexsurfer** [comment]
fixing

**Comment by flexsurfer** [comment]
fixed

**Comment by flexsurfer** [comment]
image

File: src/status_im/communities/e2e.cljs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+(ns status-im.communities.e2e
```
Comment (ilmotta) [inline src/status_im/communities/e2e.cljs:1]: Nice idea!

**Review by ilmotta — APPROVED** [review]
`remove outdated code`

Always a pleasure to see these 3 words.

**Review by ajayesivan — APPROVED** [review]

**Comment by flexsurfer** [comment]
@pavloburykh i think we could merge it without manual testing, just need your confirm , thanks

**Comment by pavloburykh** [comment]
> @pavloburykh i think we could merge it without manual testing, just need your confirm , thanks

@flexsurfer let me do a smoke check. I will ping you once I finish.

**Comment by pavloburykh** [comment]
@flexsurfer I see that "Import community" is still there. According to the description you wanted to remove it, correct me if am wrong. 

photo_2023-12-19 12 32 57

**Comment by flexsurfer** [comment]
oh you're right, leftover, so we should hide (+) button in the release build

**Comment by flexsurfer** [comment]
thanks @pavloburykh removed item, and now i hide (+) button if there is no `:FAST_CREATE_COMMUNITY_ENABLED` flag

**Comment by pavloburykh** [comment]
@flexsurfer thanx for the PR. Ready for merge.

**Pull request merged**